jobs/build/camel-k_sync/Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env groovy
+node {
+    checkout scm
+    load("pipeline-scripts/commonlib.groovy").describeJob("camel-k_sync", """
+        -----------------------------
+        Sync Camel-K client to mirror
+        -----------------------------
+        http://mirror.openshift.com/pub/openshift-v4/clients/camel-k/
+
+        Timing: This is only ever run by humans, upon request.
+    """)
+}
+
+pipeline {
+    agent any
+    options { disableResume() }
+
+    parameters {
+        string(
+            name: "VERSION",
+            description: "Desired version name. Example: 1.0",
+            defaultValue: "",
+            trim: true,
+        )
+        string(
+            name: "LINUX_BINARIES_LOCATION",
+            description: "Example: http://rcm-guest.app.eng.bos.redhat.com/rcm-guest/staging/integration/RHI-1.0.0.TP1-CR7/linux-client-1.0.0.fuse-jdk11-800042-redhat-00006.tar.gz",
+            defaultValue: "",
+            trim: true,
+        )
+        string(
+            name: "MACOS_BINARIES_LOCATION",
+            description: "Example: http://rcm-guest.app.eng.bos.redhat.com/rcm-guest/staging/integration/RHI-1.0.0.TP1-CR7/mac-client-1.0.0.fuse-jdk11-800042-redhat-00006.tar.gz",
+            defaultValue: "",
+            trim: true,
+        )
+        string(
+            name: "WINDOWS_BINARIES_LOCATION",
+            description: "Example: http://rcm-guest.app.eng.bos.redhat.com/rcm-guest/staging/integration/RHI-1.0.0.TP1-CR7/windows-client-1.0.0.fuse-jdk11-800042-redhat-00006.tar.gz",
+            defaultValue: "",
+            trim: true,
+        )
+    }
+
+    stages {
+        stage("Validate params") {
+            steps {
+                script {
+                    if (!params.VERSION) {
+                        error "VERSION must be specified"
+                    }
+                }
+            }
+        }
+        stage("Clean working dir") {
+            steps {
+                sh "rm -rf ${params.VERSION}"
+                sh "mkdir ${params.VERSION}"
+            }
+        }
+        stage("Download artifacts") {
+            parallel {
+                stage("linux")   { steps { sh "curl -sL ${params.LINUX_BINARIES_LOCATION} -o ${params.VERSION}/camel-k-client-${params.VERSION}-linux-64bit.tar.gz" }}
+                stage("macos")   { steps { sh "curl -sL ${params.MACOS_BINARIES_LOCATION} -o ${params.VERSION}/camel-k-client-${params.VERSION}-mac-64bit.tar.gz" }}
+                stage("windows") { steps { sh "curl -sL ${params.WINDOWS_BINARIES_LOCATION} -o ${params.VERSION}/camel-k-client-${params.VERSION}-windows-64bit.tar.gz" }}
+            }
+        }
+        stage("Generate MD5 sum") {
```

Comment:
MD5 is insecure and shouldn't be used anymore. Is this specially requested?

Reply:
yeah, not sure why they don't use `sha256sum` like everything else we sync ... i'm just generating them again because the file names changed, but originally they provide the checksums too. (more context on [ART-1813](https://issues.redhat.com/browse/ART-1813))